installer/vagrant/debian.sh

```diff
@@ -1,5 +1,68 @@
 #!/bin/bash
 
+showhelp() {
+    echo "Usage: ubuntu.sh [options]
```

Comment:
```suggestion
    echo "Usage: debian.sh [options]
```
You might also consider merging the ubuntu.sh and debian.sh scripts into one script, they seem to have large amounts of overlap.

Reply:
Fixed and merged